src/app/pages/checkout-review/formly/checkout-review-tac-field/checkout-review-tac-field.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  selector: 'ish-checkout-review-tac-field',
+  templateUrl: './checkout-review-tac-field.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CheckoutReviewTacFieldComponent extends FieldType {
+  formControl: FormControl;
+
+  get args() {
```

Comment:
because the args won`'t change during component life cycle you better use a regular function getArgs()